lib/resources/test/cypress/plugins/index.ts

```diff
@@ -0,0 +1,34 @@
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+const wp = require('@cypress/webpack-preprocessor')
```

Comment:
This plugin supports TypeScript, but it assumes webpack in place.

It fails with cli-bundler + systemjs + typescript setup where webpack is not available.

However, after I manually install webpack for the systemjs/ts setup. Cypress is able to run.

Reply:
Looking into it. 
Possible quick/easy solution would be to add webpack to as a dep.
Investigating if I could use the cli-bundler as a preprocessor.

Reply:
Not cli bundler, but directly use typescript compiler. It seems cypress does not have hard dep on webpack or browserify.

Reply:
Yes that's a good idea, I was just reading the typescript compiler api examples. I also noticed cypress does not have those deps although somewhere in the docs it states it is shipped with browserify...

Reply:
Added webpack & ts-loader as dep to get it working for now.